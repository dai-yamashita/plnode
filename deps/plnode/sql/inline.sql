DO $$ plnode.elog(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE plnode;
