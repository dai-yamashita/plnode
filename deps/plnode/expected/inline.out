DO $$ plnode.elog(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE plnode;
NOTICE:  this is inline code
